priv/xcodegen/Sources/TemplateApp/ContentView.swift

```diff
@@ -15,14 +15,16 @@ struct ContentView: View {
             ),
             addons: []
         ) {
-          // connecting
+            // connecting
             ProgressView()
         } disconnected: {
             DisconnectedView()
         } reconnecting: { content, isReconnecting in
-            ReconnectingView(content: content, isReconnecting: isReconnecting)
+            ReconnectingView(isReconnecting: isReconnecting) {
```

Comment:
now I'm curious why the original method didn't work

Reply:
If you're referring to the banner not being at the top of the screen, that was fixed with this modifier: https://github.com/liveview-native/liveview-client-swiftui/pull/1260/files#diff-582b7403b42572a7b5f6ee58beec2c6d043fae407237f10da7d9b157d00cede3R14

Reply:
I was curious why using the closure instead of passing the value to the named argument or is this just a style decision?

Reply:
Its a style thing, but It also makes it easier to use in the preview because now it won't force you to erase the content to an `AnyView`.